core/src/test/java/org/frankframework/core/AdapterHideRegexTest.java

```diff
@@ -79,7 +79,7 @@ public JavaListener<String> setupJavaListener(String name) {
 	private  <M> Receiver<M> setupReceiver(IListener<M> listener, String hideRegex) {
 		@SuppressWarnings("unchecked")
 		Receiver<M> receiver = configuration.createBean(Receiver.class);
-		configuration.autowireByName(listener);
+		configuration.autowireByName(receiver);
```

Comment:
😨

Reply:
Java, copy-pasta foutje maar omdat we zoveel al handmatig zetten en bean creeeren via de context maakt het blijkbaar toch weinig uit. 😬